third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -74,25 +79,38 @@ impl TestConfig {
         let mut pipeline = FunctionTargetPipeline::default();
         if path.contains("/checking/") {
             Self {
-                check_only: true,
+                type_check_only: true,
                 dump_ast: true,
                 pipeline,
                 generate_file_format: false,
+                dump_annotated_targets: false,
             }
         } else if path.contains("/bytecode-generator/") {
             Self {
-                check_only: false,
+                type_check_only: false,
                 dump_ast: true,
                 pipeline,
                 generate_file_format: false,
+                dump_annotated_targets: true,
             }
         } else if path.contains("/file-format-generator/") {
             pipeline.add_processor(Box::new(LiveVarAnalysisProcessor {}));
             Self {
-                check_only: false,
+                type_check_only: false,
                 dump_ast: false,
                 pipeline,
                 generate_file_format: true,
+                dump_annotated_targets: true,
+            }
+        } else if path.contains("/visibility-checker/") {
+            // TODO: merge this branch into "/checking/".
```

Comment:
After speaking with @wrwg, we may keep this separate, i.e., keep this code as is.

Reply:
Remove the TODO then?